src/DonorProfiles/Tabs/TabsRegister.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Give\DonorProfiles\Tabs;
+
+use Give\DonorProfiles\Tabs\Contracts\Tab;
+use http\Exception\InvalidArgumentException;
+
+class TabsRegister {
+	/**
+	 * FQCN of Tab classes
+	 *
+	 * @since 2.11.0
+	 *
+	 * @var string[]
+	 */
+	private $tabs = [];
+
+	/**
+	 * Returns all of the registered tabs
+	 *
+	 * @since 2.11.0
+	 *
+	 * @return string[]
+	 */
+	public function getTabs() {
+		return $this->tabs;
+	}
+
+	/**
+	 * Checks to see if a tab is registered with the given ID
+	 *
+	 * @since 2.11.0
+	 *
+	 * @param string $id
+	 *
+	 * @return bool
+	 */
+	public function hasTab( $id ) {
+		return isset( $this->tabs[ $id ] );
+	}
+
+	/**
+	 * Returns a tab with the given ID
+	 *
+	 * @since 2.11.0
+	 *
+	 * @param string $id
+	 *
+	 * @return string
+	 */
+	public function getTab( $id ) {
+		if ( ! isset( $this->tabs[ $id ] ) ) {
+			throw new \InvalidArgumentException( "No migration exists with the ID {$id}" );
+		}
+
+		return $this->migrations[ $id ];
```

Comment:
I'm assuming that this would be returning an item from the `$this->tabs` property?

Reply:
Ah, good catch! Yes - meant to be returning from the tabs property. Done, ref: https://github.com/impress-org/givewp/pull/5577/commits/56179f86fb9de23248a6bf0f5b2d68f8aca4e554